### PR TITLE
[STAT] History page for the pool weekly rewards emission

### PR DESCRIPTION
**Pull request by james-turner**
- Rewards history with etherscan tx links
- Dark mode support enabled

### CODE REVIEW

**Comment by james-turner** [comment]
HODL vaults accidentally fixed too 👍 
Screenshot 2021-03-15 at 22 30 00

File: src/app/static/static-values.ts [hunk 1]
```diff
@@ -141,6 +141,9 @@ export class StaticValues {
   ];
 
   public static strategiesListNonUniLpPools: string[] = [
+    'ETH_DAI_HODL',
```
Comment (belbix) [inline src/app/static/static-values.ts:144]: remove the changes from this file, please
new strategies already added on last weekends

File: src/app/dialogs/rewards-history-dialog/rewards-history-dialog.component.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                private log: NGXLogger) {
+    }
+
+    ngAfterViewInit(): void {
+        const startDate = new Date();
+        startDate.setDate(startDate.getDate() - 5); // start from 5 days ago
+        this.loadRewardsHistory(startDate, new Date());
+    }
+
+    public removePsVault = (reward: RewardDto) => reward.vault.toLowerCase() != "ps";
```
Comment (belbix) [inline src/app/dialogs/rewards-history-dialog/rewards-history-dialog.component.ts:34]: please, use for exclusion Utils.isAutoStakeVault 
sorry for unclear requirements when we discussed it

Comment (james-turner) [reply src/app/dialogs/rewards-history-dialog/rewards-history-dialog.component.ts:34]: No problem, consider it underway

File: src/app/dialogs/rewards-history-dialog/rewards-history-dialog.component.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        startDate.setDate(startDate.getDate() - 5); // start from 5 days ago
+        this.loadRewardsHistory(startDate, new Date());
+    }
+
+
+    private loadRewardsHistory(startDate: Date, endDate: Date): void {
+        // call service api here for data that is about rewards history?!?!!?
+        this.httpService.getAllHistoryRewards(Math.floor(startDate.getTime()/1000), Math.floor(endDate.getTime()/1000)).subscribe((data) => {
+            console.log(data);
+            this.rewards.push(...(data.filter(r => Utils.isAutoStakeVault(r.vault)).map(RewardDto.fillBlockDateAdopted).reverse()));
```
Comment (belbix) [inline src/app/dialogs/rewards-history-dialog/rewards-history-dialog.component.ts:40]: !Utils.isAutoStakeVault(r.vault)

**Pull request merged**